src/limbo/bayes_opt/boptimizer.hpp

```diff
@@ -30,7 +30,8 @@ namespace limbo {
             boost::parameter::optional<tag::initfun>,
             boost::parameter::optional<tag::acquifun>,
             boost::parameter::optional<tag::stopcrit>,
-            boost::parameter::optional<tag::modelfun>> boptimizer_signature;
+            boost::parameter::optional<tag::modelfun>>
+            boptimizer_signature;
```

Comment:
I'm sorry, I don't fully control clang-format. It seems to have a life of its own on some aspects. Do you validate or request changes ?